Utils/DataRetrieval.cs

```diff
@@ -0,0 +1,33 @@
+using Company.Function;
+using MySqlConnector;
+
+namespace BackEndFucntions
+{
+    public class DataRetrieval
+    {
+        public List<dynamic> RetrieveCertainType(int userID, DateTime today, string ACTIVITY_TYPE)
+        {
+            List<dynamic> Transports = null;
+            string QUERY = "SELECT FROM Activities WHERE UserID = '" + userID + "' AND DateTime = '" + today.ToString("yyyy/MM/dd") + "' AND ActivityType = '" + ACTIVITY_TYPE + "'";
```

Comment:
UserID is a foreign key, use a JOIN to validate userID exists in Users table. View GetEmissionHistory line 82 and 83 for example.